oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/NimbusJwtDecoder.java

```diff
@@ -264,6 +270,19 @@ public JwkSetUriJwtDecoderBuilder restOperations(RestOperations restOperations)
 			return this;
 		}
 
+		/**
+		 * Use the given {@link Cache} to store
+		 * <a href="https://tools.ietf.org/html/rfc7517#section-5">JWK Set</a>.
+		 *
+		 * @param jwkSetCache the {@link Cache} to be used to store JWK Set
+		 * @return a {@link JwkSetUriJwtDecoderBuilder} for further configurations
+		 */
+		public JwkSetUriJwtDecoderBuilder jwkSetCache(Cache jwkSetCache) {
```

Comment:
I'm a bit on the fence here, but I lean towards this method being called `cache` instead of `jwkSetCache`. The reason for that is the `restOperations` method isn't called `jwkSetRestOperations`.